src/Npgsql/NpgsqlConnector.cs

```diff
@@ -607,32 +607,30 @@ async Task RawOpen(NpgsqlTimeout timeout, bool async, CancellationToken cancella
                     case 'S':
                         var clientCertificates = new X509CertificateCollection();
                         var certPath = Settings.ClientCertificate ?? PostgresEnvironment.SslCert;
-                        var certPathExists = true;
 
-                        if (certPath is null)
+                        if (certPath is null && PostgresEnvironment.SslCertDefault is { } certPathDefault && File.Exists(certPathDefault))
```

Comment:
This one I forgot to do, will fix.

Reply:
Oh, I misread this. Forgot about the default of `PGSSLROOTCERT`.

The code you mentioned wasn't affected. If `certPath` has an user specified value we will use it. Otherwise, we will try to use the default value if the file exists.

Reply:
Sorry, my mistake for misreading.